template/e2e-tests/tests/landingPageTests.spec.ts

```diff
@@ -57,16 +57,16 @@ test.describe('cookie consent tests', () => {
     expect(cookieObject.categories.includes('analytics')).toBeTruthy();
 
     // Wait for Google Analytics cookies to be set after accepting
-    const MAX_TIME = 15000;
+    const MAX_TIME = 45000;
     const startTime = Date.now();
     while (cookies.length === 1) {
       if (Date.now() - startTime > MAX_TIME) {
         throw new Error('Timeout: Google Analytics cookies not set.');
       }
       cookies = await context.cookies();
     }
-
-    const gaCookiesArr = cookies.filter((c) => c.name.startsWith('_ga'));
+    let gaCookiesArr = await context.cookies(); // Call the cookies method again to avoid race condition
+    gaCookiesArr = gaCookiesArr.filter((c) => c.name.startsWith('_ga'));
     expect(gaCookiesArr.length === 2).toBeTruthy();
```

Comment:
Change this to

```suggestion
    expect(gaCookiesArr.length).toBe(2);
```

in order to get more info if test fails.